net/bind/Makefile

```diff
@@ -61,6 +61,7 @@ define Package/bind-libs
 	+libpthread \
 	+libatomic \
 	+libuv \
+	+liburcu \
```

Comment:
This is newly added dependency, would be great to mention it in the commit description, why it is required, etc. because your commit looks like e.g https://github.com/openwrt/packages/commit/1761ed0cbec0fd89e154ca3a0620d45ad3055daf , but it contains multiple changes (adding new dependency, dropping epoll), which is not wanted.

Reply:
It won't build without `liburcu`, and it no longer uses `epoll()`.

Per the `CHANGES` file:

```
6172.   [cleanup]       Refactor the loop manager and qp-trie code to remove
                        isc_qsbr and use liburcu instead. [GL #3936]

6171.   [cleanup]       Remove the stack implementation added in change 6108:
                        we are using the liburcu concurrent data structures
                        instead. [GL !7920]
...
6122.   [func]          BIND now requires liburcu for lock-free data structures
                        and concurrent safe memory reclamation. It replaces the
                        home-grown lock-free linked list and QSBR machinery
                        added in changes 6108 and 6109.  [GL #3935]
```